spec/rails_helper.rb

```diff
@@ -0,0 +1,32 @@
+ENV['RAILS_ENV'] ||= 'test'
+require 'spec_helper'
+require File.expand_path('../config/environment', __dir__)
+require 'rspec/rails'
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
+ActiveRecord::Migration.maintain_test_schema!
+RSpec.configure do |config|
+  config.use_transactional_fixtures = false
+  # config.include Devise::Test::ControllerHelpers, type: :controller
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+  end
+  config.before(:each) do
+    DatabaseCleaner.strategy = :transaction
+  end
+  config.before(:each, js: true) do
+    DatabaseCleaner.strategy = :truncation
+  end
+  config.before(:each) do
+    DatabaseCleaner.start
+  end
+  config.after(:each) do
+    DatabaseCleaner.clean
+  end
+  config.before(:all) do
+    DatabaseCleaner.start
+  end
+  config.after(:all) do
+    DatabaseCleaner.clean
+  end
+  config.infer_spec_type_from_file_location!
+end
```

Comment:
I get the following error when running rspec:
![image](https://user-images.githubusercontent.com/31193823/194343809-b23a0e3f-b970-4350-a880-929998cab97c.png)

See [this link](https://stackoverflow.com/questions/61084374/databasecleaner-fails-on-transaction) for reference. Does this issue only occur for me or does it also happen on anyone else's end? Either way, this should help solve